### ISSUE TITLE
Uploading Image

### ISSUE THREAD

**Issue by MeerZaheen** [hidjou/classsed-react-firebase-client#14]
I can upload new image from postman but I cannot upload image from web app.

![image](https://user-images.githubusercontent.com/19266944/74705671-bc30e080-51e2-11ea-9871-d82c205aec68.png)

**Comment by sshah98** [comment]
I'm getting a similar error, it's related to a CORS issue

**Comment by vip110997** [comment]
Did anyone find a solution for this?

**Comment by sshah98** [comment]
Yeah, the issue is due to the `admin.js` file and the service account associated with the **functions** folder.

Basically, in the **functions** folder, there is an `admin.js` file with the following code:
```
const admin = require("firebase-admin");
const serviceAccount = require("../db.json");
 admin.initializeApp({
   credential: admin.credential.cert(serviceAccount),
   databaseURL: "https://yourUrl.firebaseio.com",
 });
const db = admin.firestore();
module.exports = { admin, db };
```

simply remove the code related to the service account as such:
```
const admin = require("firebase-admin");
admin.initializeApp();
const db = admin.firestore();
module.exports = { admin, db };
```

This is what worked for me. Also, be careful if you're on the spark plan not to have too many requests within 100 seconds, otherwise, you'll get a CORS error.

**Comment by vip110997** [comment]
Thanks for the quick reply @sshah98. The code in my  `admin.js` is as follows

`const express = require("express");
const admin = require("firebase-admin");
admin.initializeApp();
var db = admin.firestore();
module.exports = { admin, db };` 

However, I am getting the same error
